paddle/fluid/operators/scale_op.cc

```diff
@@ -54,6 +54,22 @@ class ScaleOp : public framework::OperatorWithKernel {
     ctx->SetOutputDim("Out", ctx->GetInputDim("X"));
     ctx->ShareLoD("X", /*->*/ "Out");
   }
+
+  framework::OpKernelType GetExpectedKernelType(
+      const framework::ExecutionContext &ctx) const override {
+    auto input_data_type =
+        framework::OperatorWithKernel::IndicateVarDataType(ctx, "X");
+
+#ifdef PADDLE_WITH_MKLDNN
+    using mkldnn::memory;
```

Comment:
Maybe mkldnn::memory is not used in this function?

Reply:
You're right, thank you I have missed that

Reply:
You're right, I have missed that, thank you